docs/content/release_notes/supported_storage.md

```diff
@@ -1,18 +1,17 @@
 # Supported storage systems
 
-IBM® block storage CSI driver 1.8.0 supports different IBM storage systems as listed in the following table.
+IBM® block storage CSI driver 1.9.0 supports different IBM storage systems as listed in the following table.
 
 |Storage system|Microcode version|
 |--------------|-----------------|
-|IBM FlashSystem™ A9000|12.3.2.b or later|
-|IBM FlashSystem A9000R|12.3.2.b or later|
-|IBM Spectrum Virtualize™ family including IBM SAN Volume Controller (SVC) and IBM FlashSystem® family members built with IBM Spectrum® Virtualize (including FlashSystem 5xxx, 7200, 9100, 9200, 9200R)|7.8 and above, 8.x|
-|IBM Spectrum Virtualize as software only|7.8 and above, 8.x|
-|IBM DS8000® family|8.x and higher with same API interface|
+|FlashSystem™ A9000|12.3.2.b or later|
+|FlashSystem A9000R|12.3.2.b or later|
+|Spectrum Virtualize™ family including Spectrum Virtualize as software only, Spectrum Virtualize for Public Cloud, SAN Volume Controller (SVC), and FlashSystem® family members built with Spectrum® Virtualize (including FlashSystem 5xxx, 7xxx, 9xxx)|7.8 and above, 8.x|
+|DS8000® family|8.x and higher with same API interface|
 
 **Note:**
 
 - For the latest microcode storage support information, see the [Lifecycle and support matrix](https://www.ibm.com/docs/en/stg-block-csi-driver?topic=SSRQ8T/landing/csi_lifecycle_support_matrix.html).
```

Comment:
do we plan to add this page as a markdown file in this project as well?
because I think it could use a review a well (e.g. `FlashSystem 5xxx, 7200, 9100, 9200, 9200R` -> `FlashSystem 5xxx, 7xxx, 9xxx`)

Reply:
no. please see CSI-3747 for matrix tickets and links for review